doc/docs/master/how-tos/pipeline-operations/updating_pipelines.md

```diff
@@ -100,40 +100,45 @@ To update the code in your pipeline, complete the following steps:
 
 1. Build, tag, and push the new image to your image registry:
 
-   * If you prefer to use Pachyderm commands:
+      * If you prefer to use Pachyderm commands:
+
+         1. [Build your new image](../../developer-workflow/working-with-pipelines/#step-2-build-your-docker-image) using `docker build`. No tag needed, the folllowing [`--push-images` flag](../../developer-workflow/push-images-flag/) flag will take care of it.
```

Comment:
question on: "No tag needed"
I thought we still need a local tag for --push-images to find the right image to re-tag and push. Can you clarify what this means?

Reply:
nevermind! I see it works the way you described 👍

Reply:
Added the following example to make things a little clearer when I say "no tag needed". ?
```(for example, in a makefile: `@docker build --platform linux/amd64 -t $(DOCKER_ACCOUNT)/$(CONTAINER_NAME) .`)```